tic-tac-toe.js

```diff
@@ -1,7 +1,91 @@
-function TicTacToe() {
+$(document).ready(function () {
+  var game = new TicTacToe();
+  var cells = [1, 2, 3, 4, 5, 6, 7, 8, 9];
 
-}
+  cells.forEach(function(cell) {
+    $square = $("#cell" + cell);
+    $square.one( "click", (function () {
+      $(this).css("background-color", "#16A765");
+       var id = $(this).attr("id");
+      if (game.turns % 2 === 0) {
+        $(this).css("background-color", "#000");
+      }
+      game.play(id);
+    }));
+  });
+});
 
-TicTacToe.prototype = {
-  
+function TicTacToe() {
+  this.player = "Player 1";
+  this.turns = 1;
+  this.board = [[0, 1, 2], [3, 4, 5], [6, 7, 8]];
+  this.finish = false;
 }
+
+TicTacToe.prototype.turn = function(id) {
+  var block = id[4] - 1;
+  var row;
+  if (block < 3) {
+    row = 0;
+  } else if (block >= 3 && block < 6) {
+    row = 1;
+  } else {
+    row = 2;
+  }
+  column = block % 3;
```

Comment:
Nice way to use the block info to calculate the column. I think you have unintentionally created a global variable here without specifying `var` at the beginning